server/service/src/sync/translations/invoice.rs

```diff
@@ -298,6 +303,14 @@ impl SyncTranslation for InvoiceTranslation {
             }
         };
 
+        // If it was created after allocation, then it must be a backdated invoice
+        let backdated_datetime = match mapping.created_datetime
```

Comment:
Is this for backwards compatibility with RC build ? I don't think we need to support it

Reply:
Was thinking for mSupply backdated, sounds like better to use picked_datetime from legacy...

Reply:
[9124e6c](https://github.com/msupply-foundation/open-msupply/pull/4914/commits/9124e6c461955beee432d202b8ced76f2c01aa6f)